week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,31 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isBetween5and10(num) {
```

Comment:
You want `isBetween5and10`, `isLongName` and `startsWithD` **always** to return a boolean. What happens when you do `startsWithD("Zareen")`? How can you fix that?

Reply:
well i could just return the value and remove the if statement e.g.
function isBetween5and10(num) {
  if (num >= 5 && num <= 10) return true;
} 
could be changed to
function isBetween5and10(num) {
  return num >= 5 && num <= 10);
}